### PR TITLE
Nav-bar and smooth scroll

### PR DESCRIPTION
**Pull request by mikemai2awesome**

### CODE REVIEW

File: src/_patterns/02-components/bolt-nav-bar/src/nav-bar.scss [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  // .c-bolt-nav-list--large &
+  //   bottom: (bolt-v-spacing(large) * -1);
+  // }
+  //
+  // .c-bolt-nav-list--xlarge &
+  //   bottom: (bolt-v-spacing(xlarge) * -1);
+  // }
+}
+
+// TODO: [Mai] This is the setup for enhanced mobile view instead of hiding the items.
```
Comment (remydenton) [inline src/_patterns/02-components/bolt-nav-bar/src/nav-bar.scss]: +1 for making a note of the rationale for this being commented out.

File: src/_patterns/02-components/bolt-accordion.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Accordion
+---
+
+Example Bolt Accordion docs!
```
Comment (remydenton) [inline src/_patterns/02-components/bolt-accordion.md]: Good call getting these documentation stubs added.

As a rule of thumb, if you can submit updates like this as a separate PRs, it will make things a little easier to manage (i.e. create one PR for 'Add stub documentation' and another called 'Navbar'-- that way, if the navbar needs to get kicked back for further work, it doesn't also have to hold up the stub documentation).

File: src/_patterns/02-components/bolt-nav-bar/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+@import '@bolt/settings-global';
+@import '@bolt/settings-colors';
+@import '@bolt/settings-font-size';
+@import '@bolt/settings-font-weight';
+
+// Tools
+@import '@bolt/tools-spacing';
+@import '@bolt/tools-color-palette';
+@import '@bolt/tools-font-size';
+@import '@bolt/tools-font-weight';
```
Comment (remydenton) [inline src/_patterns/02-components/bolt-nav-bar/README.md]: Is this still accurate?  It looks to me like we may have combined these into this:
```
@import '@bolt/settings-all';
@import '@bolt/tools-all';
```

https://github.com/bolt-design-system/bolt/pull/220/commits/3a8d342b812f126c2b541c886c5dbf3dd62c12af#diff-2e0ab4481eebbb7cdd88073bb65cdc74R47

Only thing worse than no documentation is misleading/incorrect documentation...

File: src/_patterns/02-components/bolt-accordion.md (hunk 2 above)
Comment (sghoweri) [reply src/_patterns/02-components/bolt-accordion.md]: 👍🏼

File: src/_patterns/02-components/bolt-nav-bar/README.md (hunk 3 above)
Comment (sghoweri) [reply src/_patterns/02-components/bolt-nav-bar/README.md]: Yes please - at a minimum I want to start getting us in the habit of only requiring in those two Core deps (which will soon be the single core-ui package I've mentioned)

Comment (sghoweri) [reply src/_patterns/02-components/bolt-nav-bar/README.md]: @mikemai2awesome @remydenton I'm gonna merge this in to prep for me merging and releasing a new pre-release tomorrow (with my video player updates folded in) - could we make sure we get these dependency imports normalized as a separate follow-up update?
